ci-operator/step-registry/oadp/execute-tests/oadp-execute-tests-commands.sh

```diff
@@ -18,6 +18,14 @@ API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:64
 RESULTS_FILE="/alabama/cspi/e2e/junit_report.xml"
 LOGS_FOLDER="/alabama/cspi/e2e/logs"
 
+#Setup junit path for ROSA
+IS_OIDC=$(oc get authentication cluster -o jsonpath='{.spec.serviceAccountIssuer}')
```

Comment:
In case of ocp-interop it's logged in as a ServiceAccount which doesn't have permission to read this. We need to move it below the logic part.  Sorry I forgot it requires privileged access.